### PR TITLE
fix: remove on/off switch button

### LINKED ISSUE #25: Switch on/off button not implemented
https://github.com/mdn/webaudio-examples/blob/e7f0c804b4bd3c49bcbb9eaf751943133df56da1/audio-basics/index.html#L33

### PR DESCRIPTION
**Pull request by rohini-ranjanR**
### Description
Comment on/off switch button, remove the grid for the switch button make changes in the margins.

### Related issues and pull requests
Fixes #25

### CODE REVIEW

File: audio-basics/index.html [hunk 1]
```diff
@@ -52,14 +52,15 @@
 
           <label for="panner">PAN</label>
 
+          <!--
```
Comment (sideshowbarker) [inline audio-basics/index.html:55]:
```suggestion
```

File: audio-basics/index.html [hunk 2]
```diff
@@ -52,14 +52,15 @@
...
           <button
             class="control-power"
             role="switch"
             aria-checked="false"
             data-power="on"
           >
             <span>On/Off</span>
```
Comment (sideshowbarker) [inline audio-basics/index.html:62]:
```suggestion
```

File: audio-basics/index.html [hunk 3]
```diff
@@ -52,14 +52,7 @@
...
 
-          <button
-            class="control-power"
-            role="switch"
-            aria-checked="false"
-            data-power="on"
-          >
-            <span>On/Off</span>
-          </button>
+          </button>  -->
```
Comment (sideshowbarker) [inline audio-basics/index.html:55]:
```suggestion
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Rohini, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

**Comment by rohini-ranjanR** [comment]
> Rohini, thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

Thanks, Michael

**Pull request merged**